src/benchmark/operators/union_positions_benchmark.cpp

```diff
@@ -59,7 +59,7 @@ class UnionPositionsBenchmarkFixture : public benchmark::Fixture {
      */
     _referenced_table = std::make_shared<Table>();
     for (auto column_idx = 0; column_idx < num_columns; ++column_idx) {
-      _referenced_table->add_column("c" + std::to_string(column_idx), "int");
+      _referenced_table->add_column("c" + std::to_string(column_idx), TypeSymbol::Int);
```

Comment:
I find `TypeSymbol` to be very broad. This could be standing for many other things as well. Also, symbol has another meaning in C(++) which makes methods like `type_symbol_from_type` confusing.

Options that I like better: `HyriseDataTypes`. `ColumnTypes`.

Reply:
My first idea was `ColumnType` which was already taken (see above). So I was thinking about what this enum does. It indicates or represents a data type. 

Definition for “symbol”: a mark or character used as a conventional representation of an object, function, or process.

`DataTypeSymbol` I found too long. Some other ideas I had: token, indication.

Reply:
Couldn't this just be `DataType::Int`?

Reply:
I'd vote for `DataType::` as well

Reply:
And how would you call the template parameter `DataType` in `resolve_column_type`?

Reply:
`DataType`. Is that possible?